src/App.js

```diff
@@ -71,44 +72,52 @@ function App() {
     }
   }
 
+  function updateTableId(id) {
+    setTableId(id);
+  }
+
   function confirmAlert() {
     if (window.confirm("Would you like to speak to one member of our team?")) {
-      history.push("/ConfirmServiceText");
+      history.push("/confirm-service-text");
     }
   }
 
   function orderButtonHandler() {
     if (window.confirm("Would you like to confirm your order?")) {
-      history.push("/ConfirmOrderText");
+      history.push("/confirm-order-text");
+      setOrder([]);
     }
   }
 
   function billButtonHandler() {
     if (window.confirm("Would you like to receive the bill?")) {
-      history.push("/ConfirmBillText");
+      history.push("/confirm-bill-text");
     }
   }
 
   return (
     <Div100vh className="App">
       <header className="App__header">
```

Comment:
would look nice if you move the header into its own component, I think. :)

Reply:
Something I listed in my fine cut ;)